components/blocks/mdxComponentRenderer.tsx

```diff
@@ -24,10 +25,24 @@ export const componentRenderer: Components<{
     src: string;
     message: string;
   };
+  TableLayout: {
+    listofColumns: [
+      {
+        header: string;
+        hours: string;

```

Comment:
This looks like it is hardcoded to the software audit use-case.

Can you make it more generic so we can use it on other pages if necessary?

Reply:
Yes, but it's going to be a really complex schema, do you want me to spend time on it to make it more generic?

Reply:
It would be great if it was just markdown being passed to the component and it was converted to a table

You could use something like https://github.com/redbug312/markdown-it-multimd-table to make the conversion

Reply:
Done - now we can pass a markdown to the component to render a table. I used the same package but it didn't work for the multiline table so did some alterations to achieve desired list output.